### PR TITLE
Updating wallet listing criteria

### PR DESCRIPTION
**Pull request by konopkja**
Updating wallet listing criteria

### CODE REVIEW

**Comment by corwintines** [comment]
@konopkja pushed up changes for the wallet template. You can view it here: https://github.com/ethereum/ethereum-org-website/blob/konopkja-patch-2/.github/ISSUE_TEMPLATE/suggest_wallet.yaml

**Comment by konopkja** [comment]
> @konopkja pushed up changes for the wallet template. You can view it here: https://github.com/ethereum/ethereum-org-website/blob/konopkja-patch-2/.github/ISSUE_TEMPLATE/suggest_wallet.yaml

looking good

File: public/content/contributing/adding-wallets/index.md [hunk 1]
```diff
@@ -26,6 +25,12 @@ Wallets are rapidly changing in Ethereum. We've tried to create a fair framework
...
 - **Point of contact** - A point of contact for the wallet will greatly help us get accurate information when changes are made. This will keep updating ethereum.org manageable when gathering future information.
+- **EIP-1559 (type 2) transactions** - your wallet must support EIP-1559 (type 2) transactions.
+- **Good user experience** - While UX is subjective, if several core team members test the product and find it difficult to use, we reserve the right to reject the wallet and will instead provide useful suggestions to improve. This is done to protect our user base that is mostly comprised of beginners. 
+
+### Product removals {#product-removals}
+
+- **Updated information** - Wallet providers are responsible for resubmitting their wallet information every 6 months to ensure validity and relevance of provided information (even if there are no changes to their product). If the product team fails to do so, ethereum.org reserves the right to remove the project from the page.
```
Comment (minimalsm) [inline public/content/contributing/adding-wallets/index.md:33]:
```suggestion
- **Updated information** - Wallet providers are responsible for resubmitting their wallet information every 6 months to ensure validity and relevance of provided information (even if there are no changes to their product). If the product team fails to do so, ethereum.org may remove the project from the page. 
```

**Review by minimalsm — APPROVED** [review]

**Review by corwintines — APPROVED** [review]
LGTM

**Pull request merged**